benchmarks/src/std/containers/sequences/vector.bool/copy/test.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <benchmark/benchmark.h>
+//
+#include <algorithm>
+#include <functional>
+#include <random>
+#include <vector>
+
+using namespace std;
+
+static vector<bool> createRandomVector(const size_t size) {
+    static random_device rd;
+    static mt19937 gen{random_device{}()};
+    vector<bool> result(size);
+    generate_n(result.begin(), size, [] { return bernoulli_distribution{0.5}(gen); });
+    return result;
+}
+
+static void copy_block_aligned(benchmark::State& state) {
+    const vector<bool> source = createRandomVector(state.range(0));
+    vector<bool> dest(state.range(0), false);

```

Comment:
The `copy_n` benchmark extracts `const auto size` here, which would be nice to do in the `copy` and `move` benchmarks to make diffing them easier.